cs/sessions.texy

```diff
@@ -117,33 +129,39 @@ Zrušení dříve nastavené expirace docílíme metodou `removeExpiration()`. O
 Konfigurace session
 -------------------
 
-Konfigurace session musí být provedena dříve, než se začnou session používat. V aplikacích je nejvhodnější ji umístit do `bootstrap.php`.
+Konfigurace session musí být provedena dříve, než se začnou session používat. V aplikacích je nejvhodnější ji umístit do konfiguračního souboru `config.neon`.
 
 Co konfigurovat? Především je to expirace. Pokud se neprovede toto nastavení, všechny session proměnné vyexpirují v momentě zavření okna prohlížeče. Uchování session i po zavření prohlížeče se hodí například pro dlouhodobé přihlášení uživatele.
 
-/--php
-$session->setExpiration('14 days');
+/--
+nette:
+	session:
+		expiration: 14 days
 \--
 
 Na sdílených hostinzích je vhodné zvolit vlastní adresář, kam se mají ukládat soubory s relacemi:
 
-/--php
-$session->setSavePath(__DIR__ . '/sessions');
+/--
+nette:
+	session:
+		save_path: "%tempDir%/sessions"
 \--
```

Comment:
Lze používat i syntax `savePath`, `cookieDomain` atd. a asi by bylo vhodnější ji v příkladech použít, aby to ladilo s `autoStart`.